cidc_api/services/users.py

```diff
@@ -16,14 +16,13 @@
 import gcloud_client
 from models import Users, CIDCRole
 from emails import new_user_registration, confirm_account_approval
-
-from .utils import resource
+from auth import requires_auth
 
 users_api = Blueprint("users", __name__, url_prefix="/users")
 
 
 @users_api.route("/self", methods=["GET"])
-@resource("users/self")
+@requires_auth("self")
```

Comment:
AFAIK this will be transformed to 
`app.auth.authorized(allowed_roles=[], ...)`, will this allow everybody then or something? If so, can you might be add an explicit comment to the decorator doc.